recipes/empanadas.md

```diff
@@ -0,0 +1,5 @@
+# Empanadas
+
+1. Prepare the dough.
+2. Prepare the filling.
```

Comment:
```suggestion

1. Prepare the dough.
2. Prepare the filling.
3. Put the filling in the dough.
```

Reply:
Let's leave these changes for version 2.0.